### PR TITLE
Feat/overview - functionality + styling completed.

### PR DESCRIPTION
**Pull request by euniceyunjkim**
Hi team,

Please review and let me know if there are any changes that need to be made.

Note: I did make a change in our App file in order for me to be able to do the automated scroll - it was just adding a ID to the ratings & reviews component.

Thanks!

### CODE REVIEW

File: client/src/Components/App.jsx [hunk 1]
```diff
@@ -1,65 +1,57 @@
...
+import Related from './Related/index';
+import Reviews from './Reviews/index';
+import currentProducts from '../Contexts/CurProdContext';
+import currentStyle from '../Contexts/CurStyleContext';
 
 const axios = require('axios');
 
-function App({}) {
-
+function App() {
```
Comment (ceads1618) [inline client/src/Components/App.jsx:12]: Great job cleaning up the linter errors. I know they've just been such a joy!

File: client/src/Components/App.jsx [hunk 2]
```diff
@@ -1,65 +1,57 @@
...
+  const { prodID } = useParams();
 
   function defaultGrabber(items) {
-    for (let i = 0; i < items.length; i += 1) {
-      if (items[i]['default?'] === true) {
-        setCurStyle(items[i]);
-        return;
-      }
-    }
+    setCurStyle(items[0]);
```
Comment (ceads1618) [inline client/src/Components/App.jsx:19]: Great catch on simplifying code. It looks much more concise!

File: client/src/Components/Overview/ProductOverview.jsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import React, { useContext } from 'react';
-import currentProducts from '../../Contexts/CurProdContext.js';
+import currentProducts from '../../Contexts/CurProdContext';
```
Comment (ceads1618) [inline client/src/Components/Overview/ProductOverview.jsx:2]: Out of curiosity, what did removing the .js change? What error/indication did you receive to make that change?

File: server/routers/CartRouters.js [hunk 4]
```diff
@@ -10,10 +10,9 @@ CartRouters.get('/', (req, res) => {
     .catch((err) => res.send(err));
 });
 
-//double check how to pass body in
 CartRouters.post('/', (req, res) => {
   axios.post(path, req.body, { headers: { Authorization: config.TOKEN } })
-    .then((data) => res.send(data.data))
+    .then(() => res.status(201).end())
```
Comment (ceads1618) [inline server/routers/CartRouters.js:15]: Very thorough job combing through your code and making adjustments!

**Review by ceads1618 — APPROVED** [review]
I noticed the majority of photos are still showing as .png? I imagine that change is something coming later and as an optimization, isn't much of a concern right now.

Otherwise, great work! Everything is very readable, well compartmentalized, and concise. Great work Eunice!

File: client/src/Components/Overview/ProductOverview.jsx (hunk 3 above)
Comment (euniceyunjkim) [reply client/src/Components/Overview/ProductOverview.jsx:2]: Our current linter does not allow us to use file extensions in our import. Removing the file extension however caused our webpack to not recognize the files. To follow the linter guidelines and allow for our webpack to recognize the import without file extension I updated our webpack to resolve this issue.

**Pull request merged**